docs/_quarto.yml

```
@@ -2,8 +2,6 @@ project:
   type: seedcase-theme
 
 website:
-  title: "Seedcase Sprout"
-  #favicon: "/images/logo.svg"
   site-url: "https://sprout.seedcase-project.org/"
```

```suggestion
  title: "Seedcase Sprout"
  site-url: "https://sprout.seedcase-project.org/"
```

We still want the title in the website header, since that is also an accessibility thing as well as browsers use that as the tab title.